### PR TITLE
install chrome plugin 

### PR DESCRIPTION
**Pull request by jvelezpo**
* install chrome plugin by opening a browser window
* after a plugin is installed, group it with the installed ones in the UI
* Check if the plugin was installed after click install

### CODE REVIEW

File: src/editors/komodo.ts [hunk 1]
```diff
@@ -17,11 +17,11 @@ export default class Komodo extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/komodo.ts]: Why removing await from awaitable method?

File: src/editors/komodo.ts [hunk 2]
```diff
@@ -17,11 +17,11 @@ export default class Komodo extends Editor {
...
   }
 
   public async isPluginInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.pluginsDirectory());
+    return this.isDirectory(this.pluginsDirectory());
```
Comment (gandarez) [inline src/editors/komodo.ts]: Why removing await from awaitable method?

File: src/editors/netbeans.ts [hunk 3]
```diff
@@ -16,11 +16,11 @@ export default class Netbeans extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/netbeans.ts]: Why removing await from awaitable method?

File: src/editors/notepadpp.ts [hunk 4]
```diff
@@ -16,11 +16,11 @@ export default class Notepadpp extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/notepadpp.ts]: Why removing await from awaitable method?

File: src/editors/notepadpp.ts [hunk 5]
```diff
@@ -16,11 +16,11 @@ export default class Notepadpp extends Editor {
...
   }
 
   public async isPluginInstalled(): Promise<boolean> {
-    return await this.isFile(this.pluginsDirectory());
+    return this.isFile(this.pluginsDirectory());
```
Comment (gandarez) [inline src/editors/notepadpp.ts]: Why removing await from awaitable method?

File: src/editors/texstudio.ts [hunk 6]
```diff
@@ -21,7 +21,7 @@ export default class TeXstudio extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/texstudio.ts]: Why removing await from awaitable method?

File: src/editors/textmate.ts [hunk 7]
```diff
@@ -18,11 +18,11 @@ export default class TextMate extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/textmate.ts]: Why removing await from awaitable method?

File: src/editors/textmate.ts [hunk 8]
```diff
@@ -18,11 +18,11 @@ export default class TextMate extends Editor {
...
   }
 
   public async isPluginInstalled(): Promise<boolean> {
-    return await this.isDirectory(`${this.pluginsDirectory()}/WakaTime.tmplugin`);
+    return this.isDirectory(`${this.pluginsDirectory()}/WakaTime.tmplugin`);
```
Comment (gandarez) [inline src/editors/textmate.ts]: Why removing `await` from awaitable method?

File: src/editors/unity.ts [hunk 9]
```diff
@@ -16,7 +16,7 @@ export default class Unity extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/unity.ts]: Why removing `await` from awaitable method?

File: src/editors/webstorm.ts [hunk 10]
```diff
@@ -17,7 +17,7 @@ export default class WebStorm extends Editor {
   }
 
   public async isEditorInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.appDirectory());
+    return this.isDirectory(this.appDirectory());
```
Comment (gandarez) [inline src/editors/webstorm.ts]: Why removing `await` from awaitable method?

File: src/editors/xcode.ts [hunk 11]
```diff
@@ -43,7 +43,7 @@ export default class Xcode extends Editor {
 
   public async isPluginInstalled(): Promise<boolean> {
     const pluginPath = path.join(this.pluginsDirectory(), 'WakaTime.xcplugin/Contents');
-    return await this.isDirectory(pluginPath);
+    return this.isDirectory(pluginPath);
```
Comment (gandarez) [inline src/editors/xcode.ts]: Why removing `await` from awaitable method?

File: src/editors/gedit.ts [hunk 12]
```diff
@@ -30,7 +30,7 @@ export default class Gedit extends Editor {
   }
 
   public async isPluginInstalled(): Promise<boolean> {
-    return await this.isDirectory(this.pluginsDirectory());
+    return this.isDirectory(this.pluginsDirectory());
```
[the remark this answers is not in the document]
Comment (jvelezpo) [reply src/editors/gedit.ts]: https://eslint.org/docs/rules/no-return-await

[the remark this answers is not in the document]
Comment (jvelezpo) [reply src/editors/gedit.ts]: https://eslint.org/docs/rules/no-return-await

**Pull request merged**